src/app/components/top/solicitudes/formNuevaSolicitud.component.ts

```diff
@@ -215,63 +215,78 @@ export class FormNuevaSolicitudComponent implements OnInit {
     }
 
-    guardarSolicitud($event) {
-        if (this.tipoSolicitud === 'entrada') {
-            this.modelo.solicitud.organizacion = this.auth.organizacion;
-            // ------- solo solicitudes de entrada pueden ser autocitadas  ------
-            if (this.autocitado) {
-                this.modelo.solicitud.profesional = this.modelo.solicitud.profesionalOrigen;
-                this.modelo.solicitud.organizacionOrigen = this.modelo.solicitud.organizacion;
-                this.modelo.solicitud.tipoPrestacionOrigen = this.modelo.solicitud.tipoPrestacion;
-                // solicitudes autocitadas
-                this.modelo.estados = [{ tipo: 'pendiente' }];
-            }
-        } else {
-            this.modelo.solicitud.organizacionOrigen = this.auth.organizacion;
-            let reglaAplicada = this.arrayReglasDestino.find(r => r.destino.prestacion.conceptId === this.modelo.solicitud.tipoPrestacion.conceptId &&
-                r.destino.organizacion.id === this.modelo.solicitud.organizacion.id);
-            let reglaOrigen = reglaAplicada.origen.prestaciones.find(rule => { return rule.prestacion.conceptId === this.modelo.solicitud.tipoPrestacionOrigen.conceptId; });
-            if (reglaOrigen.auditable) {
-                this.modelo.estados = [{ tipo: 'auditoria' }];
+    guardarSolicitud() {
+        const params: any = {
+            estados: [
+                'auditoria',
+                'pendiente',
+                'ejecucion'
+            ],
+            paciente: this.paciente.documento,
+            prestacionDestino: this.modelo.solicitud.tipoPrestacion.conceptId
+        };
+        this.servicioPrestacion.getSolicitudes(params).subscribe(resultado => {
+            if (resultado.length) {
+                this.plex.toast('danger', `El paciente ya tiene una solicitud en curso para ${this.modelo.solicitud.tipoPrestacion.term}`);
             } else {
-                this.modelo.estados = [{ tipo: 'pendiente' }];
+                if (this.tipoSolicitud === 'entrada') {
+                    this.modelo.solicitud.organizacion = this.auth.organizacion;
+                    // ------- solo solicitudes de entrada pueden ser autocitadas  ------
+                    if (this.autocitado) {
+                        this.modelo.solicitud.profesional = this.modelo.solicitud.profesionalOrigen;
+                        this.modelo.solicitud.organizacionOrigen = this.modelo.solicitud.organizacion;
+                        this.modelo.solicitud.tipoPrestacionOrigen = this.modelo.solicitud.tipoPrestacion;
+                        // solicitudes autocitadas
+                        this.modelo.estados = [{ tipo: 'pendiente' }];
+                    }
+                } else {
+                    this.modelo.solicitud.organizacionOrigen = this.auth.organizacion;
+                    let reglaAplicada = this.arrayReglasDestino.find(r => r.destino.prestacion.conceptId === this.modelo.solicitud.tipoPrestacion.conceptId &&
```

Comment:
```suggestion
                    const reglaAplicada = this.arrayReglasDestino.find(r => r.destino.prestacion.conceptId === this.modelo.solicitud.tipoPrestacion.conceptId &&
```
- Un nombre más explicativo para 'r'

Reply:
corregido